src/shared/constants/getNextPage.constants.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * If delegated examiner test nav directly to office page instead of debrief page
+ * @param category
+ * @param delegatedTest
+ */
+export function getNextPage(category, delegatedTest) {
```

Comment:
rename 'category' to pageNameConstant or something more tailored, and delegatedTest: boolean

Reply:
renamed and added type